src/tabs/power.html

```diff
@@ -207,3 +210,28 @@
         </div>
     </div>
 </div>
+
+<div class="tab-power" id="calibrationmanagercontent" style="display:none; width:360px;">
```

Comment:
This div and some of the elements that contain, have a lot of "style" attributes. Please take out them to the CSS. We are trying to not add more of this to the code.

Reply:
I removed most of it, so here it is.